EssentialApp/EssentialAppTests/Helpers/ImageCommentsUIIntegrationTests+Assertions.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  ImageCommentsUIIntegrationTests+Assertions.swift
+//  EssentialAppTests
+//
+//  Created by Sebastian Vidrea on 09.04.2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeed
+import EssentialFeediOS
+
+extension ImageCommentsUIIntegrationTests {
+	func assertThat(_ sut: ListViewController, isRendering imageComments: [ImageComment], file: StaticString = #filePath, line: UInt = #line) {
+		sut.view.enforceLayoutCycle()
+
+		guard sut.numberOfRenderedImageCommentViews() == imageComments.count else {
+			return XCTFail("Expected \(imageComments.count) image comments, got \(sut.numberOfRenderedImageCommentViews()) instead.", file: file, line: line)
+		}
+
+		let viewModel = ImageCommentsPresenter.map(imageComments)
+
+		imageComments.enumerated().forEach { index, imageComment in
+			assertThat(sut, hasViewConfiguredFor: viewModel, at: index, file: file, line: line)
+		}
```

Comment:
The `imageComment` within the forEach block is never used. 

So it could iterate directly on the `viewModel.imageComments` - this way, it wouldn't need to access the values at index in the assertions.

```suggestion
		let viewModels = ImageCommentsPresenter.map(imageComments).imageComments

		viewModels.enumerated().forEach { index, viewModel in
			assertThat(sut, hasViewConfiguredFor: viewModel, at: index, file: file, line: line)
		}
```